### PR TITLE
Storing target cluster-name under SHARED_DIR

### PR DESCRIPTION
**Pull request by omrirh**

### CODE REVIEW

**Comment by omrirh** [comment]
Hello @yasun1 ,
let me know if those changes can be approved as this blocks some of our jobs.
Thanks in advance!

File: ci-operator/step-registry/osd-ccs/cluster/provision/aws/osd-ccs-cluster-provision-aws-commands.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 set -o nounset
 set -o errexit
 set -o pipefail
+set -o verbose
```
Comment (yasun1) [inline ci-operator/step-registry/osd-ccs/cluster/provision/aws/osd-ccs-cluster-provision-aws-commands.sh:6]: Let us close the verbose model.

Comment (omrirh) [reply ci-operator/step-registry/osd-ccs/cluster/provision/aws/osd-ccs-cluster-provision-aws-commands.sh]: Done

**Pull request merged**